src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const TodoForm = props => {
+    return (
+        <div>
+            <form onSubmit={props.formHandler} className="todo-menu">
+                <input
+                    name="task"
+                    value={props.task}
+                    onChange={props.inputHandler}
+                    placeholder="Todo Text"
+                />
+                <button type="submit">Create</button>
+            </form>
+            <button onClick={props.clearHandler}>Clear</button>
```

Comment:
You can put the clear button back in the form now. It's doesn't affect your form submission after all.